deploy-manage/deploy/elastic-cloud/differences-from-other-elasticsearch-offerings.md

```diff
@@ -1,81 +1,173 @@
+| **Scripted metric aggregations** | ✅ | ❌ | Not supported in Serverless |
+| **Search applications** | UI and APIs | Maintenance mode (beta), API only | UI not available in Serverless |
+| **Shard management** | User-configurable | Managed by Elastic | No manual shard allocation in Serverless |
+| **Watcher** | ✅ | ❌ | Use **Kibana Alerts** instead, which provides rich integrations across use cases |
+| **Web crawler** | ✅ (discontinued with Enterprise Search in 9.0) | Self-managed only | Use [**self-managed crawler**](https://github.com/elastic/crawler) |
+
+### Observability
+
+This table compares Observability capabilities between Elastic Cloud Hosted deployments and Serverless projects:
+
+| **Feature** | Elastic Cloud Hosted | Serverless Observability projects | Serverless notes |
+|---------|----------------------|-----------------------------------|------------------|
+| **Fleet Agent policies** | ✅ | ✅ | |
+| **APM integration** | ✅ | ✅ | Use **Managed Intake Service** (supports Elastic APM and OTLP protocols) |
+| **APM Agent Central Configuration** | ✅ | ❌ | Not available in Serverless |
+| **APM Tail-based sampling** | ✅ | ❌ | - Not available in Serverless <br>- Consider **OpenTelemetry** tail sampling processor as an alternative |
+| **Android agent/SDK instrumentation** | ✅ | ❌ | Not available in Serverless |
+| **AWS Firehose integration** | ✅ | ✅ | |
+| **Custom roles for Kibana Spaces** | ✅ | **Planned** | Coming in a future release |
+| **Data stream lifecycle** | ✅ | ✅ | Primary lifecycle management method in Serverless |
+| **Elastic Serverless Forwarder** | ✅ | ❌ | |
+| **Elastic Synthetics Private Locations** | ✅ | ✅ | |
+| **Fleet server** | - Self-hosted <br>- Hosted | ✅ | Fully managed by Elastic |
+| **Index lifecycle management** | ✅ | ❌ | Use **data stream lifecycle** instead |
+| **iOS agent/SDK instrumentation** | ✅ | ❌ | Not available in Serverless |
+| **Kibana Alerts** | ✅ | ✅ | |
+| **LogsDB index mode** | ✅ | ✅ | - Reduces storage footprint <br> - Enabled by default <br>- Cannot be disabled |
+| **Logs management** | ✅ | ✅ | |
+| **Metrics monitoring** | ✅ | ✅ | |
+| **Observability SLO** | ✅ | ✅ | |
+| **Real User Monitoring (RUM)** | ✅ | **Planned** | Coming in a future release |
+| **Universal Profiling** | ✅ | ❌ | Not available in Serverless |
+| **Uptime monitoring** | ❌ | ❌ | - Deprecated in all deployment types <br>- Use **Synthetics app** instead |
+
+### Security
+
+This table compares Security capabilities between Elastic Cloud Hosted deployments and Serverless projects:
+
+| **Feature** | Elastic Cloud Hosted | Serverless Security projects | Serverless notes |
+|---------|---------------------|------------------------------|------------------|
+| **API keys** | ✅ | ✅ | |
+| **Defend for Containers integration** | ✅ (deprecated in 9.0) | ❌ | Not supported in Serverless architecture |
+| **Endpoint security** | ✅ | ✅ | |
+| **Field and document level security** | ✅ | ✅ | |
```

Comment:
Sorry one more suggestion - I _think_ you can remove this too. Elastic Security features mainly focus on monitoring and securing your environment. This entry seems more related to securing your data within Elasticsearch. 

```suggestion
```